code/modules/mob/grab/grab_datum.dm

```diff
@@ -4,7 +4,7 @@
 	var/decl/grab/downgrab						// The grab that this will downgrade to if it downgrades, null means break grab on downgrade
 	var/stop_move = 0							// Whether or not the grabbed person can move out of the grab
 	var/reverse_facing = 0						// Whether the person being grabbed is facing forwards or backwards.
-	var/can_absorb = 0							// Whether this grab state is strong enough to, as a changeling, absorb the person you're grabbing.
+	var/can_absorb = 0							// Whether this grab state is strong enough to absorb the person you're grabbing. Deprecated.
```

Comment:
Why not remove this variable entirely? It's entirely unused outside of `/mob/proc/changeling_absorb_dna()`.